easybuild/easyconfigs/g/GDB/GDB-8.3-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDB'
+version = '8.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.gnu.org/software/gdb/gdb.html'
+description = "The GNU Project Debugger"
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['802f7ee309dcc547d65a68d61ebd6526762d26c3051f52caebe2189ac1ffd72e']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('texinfo', '6.6'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.1'),
+    ('expat', '2.2.6'),
+    ('Python', '3.7.2'),
```

Comment:
can you use multi_deps instead?
`multi_deps = {'Python': ['3.7.2', '2.7.15']}`

Reply:
No, it's not a python binding, it's built-in python, so it links the gdb binary to either libpythonX.so library, so there can only be one per PATH.